web/views/permissions_view.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Cog.V1.PermissionView do
+  use Cog.Web, :view
+
+  def render("show.json", %{permission: permission}) do
+    %{permission: %{id: permission.id,
+                    name: permission.name,
+                    namespace: permission.namespace.name} }
+  end
+
+  def render("index.json", %{permissions: permissions}) do
+    %{permissions: Enum.map(permissions, fn(permission) ->
+        %{id: permission.id,
+          name: permission.name,
+          namespace: permission.namespace.name}
+    end)}
+  end
```

Comment:
Actually on second thought, it might actually be pretty nice, since you can easily use it with `render_one` and `render_many`. Here's some untested code:

```
def render("permission.json", %{permission: permission}) do
  %{id: permission.id,
    name: permission.name,
    namespace: permission.namespace.name}
end

def render("index.json", %{permissions: permissions}) do
  %{permissions: render_many(permissions, __MODULE__, "permission.json")}
end

def render("show.json", %{permission: permission}) do
  %{permission: render_one(permission, __MODULE__, "permission.json")}
end
```